tracee/tracee.go

```diff
@@ -181,50 +181,56 @@ func (t *Tracee) initBPF(ebpfProgram string) error {
 
 	sysPrefix := bpf.GetSyscallPrefix()
 	for _, e := range eventsToTraceFinal {
-		if eName, isSyscall := EventsSyscalls[e]; isSyscall {
-			kp, err := t.bpfModule.LoadKprobe(fmt.Sprintf("syscall__%s", eName))
+		event, ok := EventsIDToEvent[e]
+		if !ok {
+			continue
+		}
+		if event.AttachMechanism == SYSCALL {
```

Comment:
not a big deal but better use a switch statement here

Reply:
This is not a switch case. If AttachMechanism is KPROBE_KRETPROBE i want to of these cases to happen